metrics/privacy_metrics/dcr_nndr.py

```diff
@@ -282,13 +282,12 @@ def _prepare_data_for_privacy_metrics(
             )
 
             # standardize
+            syn_data_p[column_name] = (
+                syn_data_p[column_name] - tgt_data_p[column_name].mean()
+            ) / np.max([tgt_data_p[column_name].std(), smoothing_factor])
```

Comment:
Could you add a comment to understand why you standardize synthetic data with target data?